packages/@aws-cdk/aws-codedeploy/lib/ecs/deployment-group.ts

```diff
@@ -146,6 +146,13 @@ export interface EcsDeploymentGroupProps {
    */
   readonly alarms?: cloudwatch.IAlarm[];
 
+  /**
+   * Whether to enable the alarm configuration.
+   *
+   * @default true if the deployment group is associated with alarms, otherwise false
+   */
+  readonly alarmsEnabled?: boolean;
+
```

Comment:
why does this need to be user-facing? The linked issue states that the alarms aren't being disabled when the alarms are finished. Can we just render alarms as disabled when there are no alarms?

Reply:
Removed